public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ConfigEditor.test.tsx

```diff
@@ -1,46 +1,63 @@
+import { render, screen } from '@testing-library/react';
 import React from 'react';
-import { shallow } from 'enzyme';
```

Comment:
curious, are we avoiding `shallow` (or `enzyme`) for any reason?

Reply:
react-testing-library is the future 😎